pyproject.toml

```diff
@@ -51,6 +51,7 @@ mashumaro = "^3.0.1"
 dill = "^0.3.4"
 pydantic = "^2.0.0"
 prometheus_client = "^0.16.0"
+numpy = "^1.26.4"
```

Comment:
Adding this version which seems to be needed for `aligned`
Unsure if version >=2 will work.